kfs-core/src/main/java/edu/arizona/kfs/pdp/businessobject/ShippingHeader.java

```diff
@@ -57,7 +68,20 @@ public void setOpenCustomField(String masterEDINumber) {
 		
     	map.put("creationDate", getCreationDate().toString());
     	map.put("transactionRefNumber", getTransactionRefNumber());
-        return map;
+    	return map;
+    }
+
+    public List<ShippingInvoice> getInvoices() {
+        if ( !invoicesLoaded ) {
+            invoices = new ArrayList<ShippingInvoice>( SpringContext.getBean(BusinessObjectService.class).findAll(ShippingInvoice.class) );

```

Comment:
Can we use injection for BusinessObjectService instead of finding it in the SpringContext?